app/scripts/modules/core/src/pipeline/config/stages/managed/ImportDeliveryConfigExecutionDetails.tsx

```diff
@@ -6,6 +6,43 @@ import { SETTINGS } from 'core/config';
 
 const NOT_FOUND = 'Not found';
 
+export class DeliveryConfigImportErrorDetails {
```

Comment:
@luispollo let's meet up in person and chat through how this works/how you want it to work — I see what you're trying to do and I think you're starting down the right general direction, but I also get the sense you _might_ be applying some of the implicit behavior you'd expect to see in Java-land around automatically turning data structures into classes with methods on them. 

I think it's easier to explain how this would work in JS/TS if we do it verbally vs. trying to write out the behavior in a PR comment.

Reply:
Sure. I'll be in the office on Thursday, let's meet then.